components/brave_new_tab_ui/apiEventsToStore.ts

```diff
@@ -37,3 +37,19 @@ export default function wireApiEventsToStore () {
     console.error('New Tab Page fatal error:', e)
   })
 }
+
+chrome.braveRewards.onWalletInitialized.addListener((result: any) => {
```

Comment:
please replace any with `result: NewTab.RewardsResult` as this is what action then uses

Reply:
Unfortunately since this is typed in `chromel.d.ts` as `RewardsExtension.Result` is what it expects, it will not coerce types to `NewTab.RewardsResult`, and I cannot bring in the namespace.

I've compromised by updating this to `result: any | NewTab.RewardsResult`